swagger/docs.go

```diff
@@ -141,6 +141,96 @@ const docTemplate = `{
                 }
             }
         },
+        "/v2/_zot/ext/mgmt": {
+            "get": {
+                "description": "Get current server configuration",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "summary": "Get current server configuration",
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/extensions.StrippedConfig"
+                        }
+                    },
+                    "500": {
+                        "description": "internal server error\".",
+                        "schema": {
+                            "type": "string"
+                        }
+                    }
+                }
+            }
+        },
+        "/v2/_zot/ext/userprefs": {
+            "put": {
+                "description": "Add bookmarks/ stars info",
```

Comment:
We can write it like this I think:
```suggestion
                "description": "Add bookmarks/stars info",
```